File: tests/fixtures/adapters/csharp/sample.cs
namespace Acme.Tools;

public interface IRunner
{
    string Run(string input);
}

public enum Mode
{
    Fast,
    Slow,
}

public record Result(bool Ok);

public class Service
{
    public string Name { get; init; }

    public event EventHandler? Changed;

    public Service(string name)
    {
        Name = name;
    }

    public async Task<string> RunAsync(string input)
    {
        return $"{Name}:{input}";
    }

    public static Service Build(string name)
    {
        return new Service(name);
    }
}

File: tests/fixtures/adapters/java/sample.java
package com.example.service;

public interface Runner {
    String run(String input);
}

public enum Mode {
    FAST,
    SLOW
}

public record Result(boolean ok) {}

public class Service {
    private final String name;

    public Service(String name) {
        this.name = name;
    }

    public String run(String input) {
        return name + ":" + input;
    }

    private static int parse(int value) {
        return value + 1;
    }
}
